samples/app-anonymous-users/csharp/assets/sample.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-app-anonymous-users-csharp",
+    "source": "officeDev",
+    "title": "Anonymous User Support",
+    "shortDescription": "This sample shows Anonymous Users Support In Meeting.",
```

Comment:
This sample app shows anonymous users support in teams meeting apps.

Reply:
Fixed